src/utils/auth.ts

```diff
@@ -0,0 +1,3 @@
+// eslint-disable-next-line import/prefer-default-export
+export const isLocalAuthEnabled = () =>
+  process.env.REACT_APP_LOCAL_AUTH === 'enabled';
```

Comment:
Is it possible to store the environment variable as a boolean? I do not know.

Reply:
I think it has to be a string, but we can choose any string we want. I just picked `enabled`, but it could be `on`, `active`... really anything. WDYT?

Reply:
I was wrong, actually: we can use any of the values here: https://golang.org/pkg/strconv/#ParseBool

I will make this change.